src/main/java/org/quiltmc/Meta.java

```diff
@@ -99,13 +99,13 @@ private void populateMappings()  {
         }
     }
 
-    private void populateYarn() {
```

Comment:
once my PR for the buildscript is done, could we also have this include the hashed version that the current QM version is built on?

Reply:
Yeah, it shouldn't be hard to do

Reply:
This can probably be done now